### PR TITLE
Removes Cryostasis Bags

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Cryo bags are nearly unused, unobtainable, very snowflakey, anddd they're very much an "easy mode" object.

If we want to have medical focus on treating patients and making tough decision, having a universal "get out of jail free" card is not the way to do it.

stasis prevents organs bleeding, organs, and a ton of other (potentially negative) processes from running.

The consequences for putting someone in stasis are nearly non-existent.

And again, the snowflakeeee.

Code
- Removed some unneeded vars, and optimized decay a bit

:cl: Fox McCloud
del: removes cryostasis bag
/:cl:

### CODE REVIEW

**Comment by SamHPurp** [comment]
"Improve don't remove".

During highly stressful situations like Warops its really nice to be able to press 'pause' on one of your patients.

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -1090,11 +1058,11 @@
 		makeSkeleton()
 		return //No puking over skeletons, they don't smell at all!
 
+	if(!isurf(loc))
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/life.dm]: isurf

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -32,8 +32,6 @@
 
 	create_reagents(330)
 
-	prev_gender = gender // Debug for plural genders
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/human.dm]: This is still an incredibly rare bug, I believe

File: code/modules/mob/living/carbon/human/life.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/life.dm]: HUE

**Comment by Anticept** [comment]
On colonial marines, these cause clone damage the longer they are in them. Clone damage on CM is very difficult to treat, so a long period of time is just as bad as letting them die.

File: code/modules/mob/living/carbon/human/human.dm (hunk 2 above)
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/human.dm]: `prev_gender` isn't even used.

**Comment by Fox-McCloud** [comment]
@Anticept 

They cause clone damage here, too.

**Comment by ktoma36** [comment]
I cant remember, do stasis bags put the progress of a xeno embryo on hold? I know they do in colonial marines. Also, why not make it something R&D can produce or a package available to cargo. In fact why don't medical ERT spawn with one or two stasis bags, they are supposed to be some of the most trained and high tech forces Cencom can send (besides deathsquads of course)

**Comment by GunDOSMk1** [comment]
These are required in rounds where you have alotta patients, and zero doctors. We need these.

**Comment by SigholtStarsong** [comment]
If anything, we need MORE of these.

**Comment by Anticept** [comment]
@Fox-McCloud right, but i was more pointing out that it would be helpful if these had a bad drawback.

Yeah they are snowflake... but in dire times they are quite useful.

**Comment by outbools** [comment]
D O N T

**Comment by Taaketa** [comment]
Never see the damn things get used and when they do, they get used for about 20 seconds till some jackass opens them

**Comment by alexkar598** [comment]
Maybe Id lock them
To doctors?

**Comment by IcyV** [comment]
The root cause of cryo bags being unused at this point in time is that medical is incredibly easy and absurdly forgiving on Para. Cloning and defibs make worrying about death a joke. There are multiple chems  to deal with basically every possible type of damage faster than most sources can deal them. Even pain is largely a non-issue most of the time. Hell, if chemistry has half a brain they can basically outfit the Paramedic so well that a patient won't ever need to even go to medbay.

Cryo bags as a thing are still useful, assuming medical and chems took massive nerfs or aren't staffed. 

I think they're just common enough to justify not removing them but it's definitely a small line.

**Comment by SirBloodBuske** [comment]
I use these almost every round, the hell

**Comment by Spartan6** [comment]
Why not nerf instead of remove..?

File: code/modules/mob/living/carbon/human/human.dm (hunk 2 above)
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/human.dm]: It's exactly as the comment says - A debugging tool, it lets you see what their gender was before it ended up bugged.

Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/human.dm]: Right. You want me to re-add it or is this edge case enough that it's not worth really keeping? I've...not really ever heard of someone having a gender they're not supposed to have, before.

**Comment by Jovaniph** [comment]
The moment I saw "Remove" Something something, I knew everyone was going to bitch and complain.

I don't know if there ever was any PR that was open/rejected with cryo-stasis bag. Was there any other solution to improve it? If no, then why complain about it being removed if its rarely used? Maybe my eyes have deceive me, but I haven't seen anyone making any improvements/nerfs on those things.

Can we stop being a bunch of pack rats with Paradise and allow things in the game to be remove or are we just going to be stuffing it while other things become useless over time?

**Comment by EvadableMoxie** [comment]
Do you find it odd that it's somehow simultaneously never used and yet 'easy mode?' Don't you think doctors would use them if they were actually good?

That's because it's not easy mode.  Status bags are rarely used because they have an incredibly narrow niche in which they are useful. They allow you to deal brain and genetic damage to someone in order to delay their dealth.  That is rarely ever useful since there's almost never a situation where an equipped medical professional can't easily stabilize someone, and even when they can't, if you reached the patient while they were still alive you'll get to Medbay in time to defib them, which is a hell of less easier than giving them genetic damage.

The only time stasis bags are good is when the shuttle is coming and you don't have time to do surgery and you just need to keep someone alive for another 3 minutes, a situation I see come up maybe 1 in every 20 rounds.

You can remove them if you want because most people won't even notice they are gone. However, I wholeheartedly disagree with the reasoning.

**Comment by tigercat2000** [comment]
@EvadableMoxie The reasoning is far more about their maintainability than balance- It's terribly coded and snowflaked all over humans for, exactly what you say, an incredibly niche usage.

**Comment by SamHPurp** [comment]
How about a controversial tag?

File: code/modules/mob/living/carbon/human/human.dm (hunk 2 above)
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/human.dm]: Naaaah, it's probably fine.

**Comment by SamHPurp** [comment]
I've had multiple people ask me if I'm going to submit a revert for this.

Submitting a reversion would mean that I have trust in the system, and I don't.

**Comment by SigholtStarsong** [comment]
This reflects very poorly on the Staff and Server as a whole.

**Comment by Spartan6** [comment]
Overwhelming thumbs down yet merged.

Hah.

**Comment by Jovaniph** [comment]
Maintainers don't decide what gets merged or not. At least for most of the PRs like this one

**Comment by shazbot194** [comment]
The problem @Jovaniph, there's is nothing that tells us why this was merged. It is both very unpopular, and reverts a feature, rather then fixing, improving, or making un-accessible. From what we, the community can tell, it was merged by the maintainers, and people aren't really happy with the resaults, and the lack of transparency is making it worse.

**Comment by Spacemanspark** [comment]
Wait why was this merged

**Comment by Jovaniph** [comment]
Why was this merged? I think its fairly obvious based on the OP.

**Comment by outbools** [comment]
holy fuck
I often play paramedic and these bags was must-have to me...
why lisa why
YOU ARE TEARING ME APART



oh hai doggie

File: code/game/objects/items/bodybag.dm [hunk 3]
```diff
@@ -89,51 +89,4 @@
 	// Make it possible to escape from bodybags in morgues and crematoriums
 	if(loc && (isturf(loc) || istype(loc, /obj/structure/morgue) || istype(loc, /obj/structure/crematorium)))
 		if(!open())
-			to_chat(user, "<span class='notice'>It won't budge!</span>")
```
Comment (outbools) [inline code/game/objects/items/bodybag.dm]: uh okay i guess

**Pull request merged**